headers/types/common/common.h

```diff
@@ -795,11 +795,13 @@ struct mission {
     int description_id;
     uint8_t unique_map_id; // 0xC: for challenges/treasure hunts/certain outlaws
     undefined field_0xd;
-    struct monster_id_16 client;                // 0xE
-    struct monster_id_16 target;                // 0x10
-    struct monster_id_16 outlaw_backup_species; // 0x12
-    struct item_id_16 item_wanted;              // 0x14
-    struct mission_reward_type_8 reward_type;   // 0x16
+    struct monster_id_16 client; // 0xE
+    // 0x10: Repurposed for storing the second team member of a normal challenge mission
```

Comment:
This wording kind of de-emphasizes the primary use of the field, which is confusing. Can we change the wording of all these comments (in `struct mission` and `struct mission_rescue_bin`) to "For challenge letter missions, ..."?